### DISCUSSION TITLE
How to add text before the form for providing details?

### DISCUSSION THREAD

**Discussion by PrasadChinwal** [filamentphp/filament#11861] [Help] [answered]
### Package

Form builder

### Package Version

v3.0.0

### How can we help you?

How can I add text before the form begins? Example:

<img width="388" alt="Screenshot 2024-03-14 at 4 09 39 PM" src="https://github.com/filamentphp/filament/assets/7385673/36fc74f2-c96c-4ef6-8f70-bbed02464769">

I want to add a list of requirements that the user needs to check before the user begins filling form. 
I have tried `ViewField` and `Placeholder` but they don't display the list. They both just output plain string.

**Comment by cirrion-software** [comment, marked as answer]
You could use a placeholder with HTML inside it:

[https://filamentphp.com/docs/3.x/forms/layout/placeholder#rendering-html-inside-the-placeholder](url)

**Comment by leandrocfe** [comment]
```php
ViewField::make('requirements')
    ->view('requirements')
```

```html
<!-- resources/views/requirements.blade.php -->
<div>
    <ul class="list-inside list-disc">
        <li>Requirement 1</li>
        <li>Requirement 2</li>
        <li>Requirement 3</li>
    </ul>
</div>
```

**Comment by PrasadChinwal** [comment]
Both solutions worked for me! I made a mistake of not adding tailwind classes to style the ul element. 

Thank you for the help!
